chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+It is at this stage that the raw data is transformed into analysis data.
+This is done by creating derived variables
+(binaries, indices, and interactions, to name a few).
+To understand why construction is necessary,
+let's take the example of a household survey's consumption module.
+It will result in separate variables indicating the 
+amount of each item in the bundle that was consumed.
+There may be variables indicating the cost of these items.
+You cannot run a meaningful regression on these variables. 
+You need to manipulate them into something that has \textit{economic} meaning. 
+During this process, the data points will typically be reshaped and aggregated 
+so that level of the data set goes from the unit of observation in the survey to the unit of analysis.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Unit\_of\_Observation}} 
+To use the same example, the data on quantity consumed was collect for each item, and needs to be aggregated to the household level before analysis.
+
+% Why it is a separate process -------------------------------
+
+% From cleaning
+Construction is done separately from data cleaning for two reasons. 
+The first one is to clear differentiation between the data originally collected and the result of data processing decisions.
+The second is to ensure that variable definition is consistent across data sources. 
+Unlike cleaning, construction can create many outputs from many inputs. 
+Let's take the example of a project that has a baseline and an endline survey. 
+Unless the two instruments are exactly the same, which is unlikely,  the data cleaning for them will require different steps, and therefore will be done separately. 
+However, you still want the constructed variables to be calculated in the same way, so they are comparable.
+So you want to construct indicators for both rounds in the same code, after merging them.
+
+% From analysis
 Data construction is never a finished process.
-It comes ``before'' data analysis only in a limited sense:
-the construction code must be run before the analysis code.
-Typically, however it is the case that the construction and analysis code
-are written concurrently -- both do-files will be open at the same time.
-It is only in the process of writing the analysis
-that you will come to know which constructed variables are necessary,
-and which subsets and alterations of the data are desired.
-
-You should never attempt to create a ``canonical'' analysis dataset.
-Each analysis dataset should be purpose-built to answer an analysis question.
-It is typical to have many purpose-built analysis datasets:
+It comes ``before'' data analysis only in a limited sense: the construction code must be run before the analysis code.
+Typically, however, construction and analysis code are written concurrently.
+As you write the analysis, different constructed variables will become necessary, as well as subsets and other alterations to the data.
```

Comment:
*Ideally* indicator construction can be done first (by following a PAP) - but in practice it's almost never that neat

Reply:
Phrased it differently in 17174e8